src/main/groovy/com/bmuschko/gradle/docker/services/DockerClientService.java

```diff
@@ -0,0 +1,108 @@
+package com.bmuschko.gradle.docker.services;
```

Comment:
We'll want to give an indication that it is not meant to be part of the plugin's public API. Can you move this into `com.bmuschko.gradle.docker.internal.services`?

Reply:
Moved